control-plane/stor-port/src/types/v0/store/volume.rs

```diff
@@ -643,6 +652,23 @@ impl PublishOperation {
         self.publish_context.clone()
     }
 }
+/// SetVolumeProperty Operation parameters.
+#[derive(Serialize, Deserialize, Debug, Clone, PartialEq)]
+pub struct VolumeProperties {
+    properties: HashMap<String, String>,
```

Comment:
We can use actual types here rather than String?

Reply:
Currently idea is to keep each attribute with its datatype in open api spec and map it to generic string based map.
VolumeAttr enum will match the name for matching the attribute to set it in Volumespec.

Reply:
fixed.